src/helpers/terminal_helpers.sh

```diff
@@ -3,6 +3,19 @@
 # Helper methods for writing to the terminal.
 
 
+# http://stackoverflow.com/a/12202793/3418047
```

Comment:
No, that's not necessary. The copyright for that source code is actually with StackOverflow. So I suggest you create your own code based on the ideas outlined in the article, and remove the link.